Misc/NEWS.d/next/Documentation/2019-05-29-13-28-01.bpo-19184.VAi7wG.rst

```diff
@@ -0,0 +1,3 @@
+Update the documentation of :mod:`dis`:
```

Comment:
This is a minor documentation change. We don't need to document it in a NEWS file. We only require NEWS entries for major documentation changes.

Reply:
Ah, ok. I didn't know that. Thank you for the review.